app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+require 'api_entity'
+class User
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ApiEntity
+
+  set_singular_path 'user/users'
```

Comment:
This is a bit unusual for a path in the backend but I must have missed that PR/not noticed

Reply:
This will never work in a review app thinking about it

The review app relies on publicly routable ALB paths to avoid needing access to internal networks.

Currently the only externally routable paths in the ALB to the backend under under `/api`

Not sure if this is an actual issue since you can always deploy to development by just adding the `needs-deployment` label